tests/run_tests.inp

```diff
@@ -0,0 +1,81 @@
+set verbose off
+clear
+
+set assert stop
+include assertion.gfn
+include "./src/candlesticks.inp" --force
```

Comment:
Misstyped path
`include "../src/candlesticks.inp" --force`

Reply:
Not really, the test is executed within the main folder `./candlesticks` by means of the command `./run_tests.sh` on a LINUX machine.
You can execute the script locally but then you would have to set the working directory. For instance, by adding to the file `./tests/run_tests.inp` file.
~~~
set workdir "PATH/candelsticks"
include "../src/candlesticks.inp" --force
~~~

However, the reason to go via the shell script is to run automatic tests in a later stage (continuous integration).